deploy/role.yaml

```diff
@@ -14,6 +14,13 @@ rules:
   - events
```

Comment:
This is a copy of what's in https://github.com/aerogear/app-metrics-operator/pull/2